File: tang-app/src/main/java/com/tang/app/entity/AppFriend.java
package com.tang.app.entity;

import com.tang.commons.base.entity.BaseEntity;

/**
 * 用户好友实体类 app_friend
 *
 * @author Tang
 */
public class AppFriend extends BaseEntity {

    @java.io.Serial
    private static final long serialVersionUID = 3418336663574009608L;

    /**
     * 用户好友ID
     */
    private Long userFriendId;

    /**
     * 用户ID
     */
    private Long userId;

    /**
     * 好友ID
     */
    private Long friendId;

    /**
     * 唯一标识
     */
    private Long uniqueId;

    /**
     * 用户名
     */
    private String username;

    /**
     * 昵称
     */
    private String nickname;


    public Long getUserFriendId() {
        return userFriendId;
    }

    public void setUserFriendId(Long userFriendId) {
        this.userFriendId = userFriendId;
    }

    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }

    public Long getFriendId() {
        return friendId;
    }

    public void setFriendId(Long friendId) {
        this.friendId = friendId;
    }

    public Long getUniqueId() {
        return uniqueId;
    }

    public void setUniqueId(Long uniqueId) {
        this.uniqueId = uniqueId;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getNickname() {
        return nickname;
    }

    public void setNickname(String nickname) {
        this.nickname = nickname;
    }

}

File: tang-app/src/main/kotlin/com/tang/app/entity/AppFriend.kt
package com.tang.app.entity

import com.tang.commons.base.entity.BaseEntity

/**
 * 用户好友实体类 app_friend
 *
 * @author Tang
 */
class AppFriend : BaseEntity() {

    companion object {
        @java.io.Serial
        private val serialVersionUID = 1L
    }

    /**
     * 用户好友ID
     */
    var userFriendId: Long? = null

    /**
     * 用户ID
     */
    var userId: Long? = null

    /**
     * 好友ID
     */
    var friendId: Long? = null

    /**
     * 唯一标识
     */
    var uniqueId: Long? = null

    /**
     * 用户名
     */
    var username: String? = null

    /**
     * 昵称
     */
    var nickname: String? = null

}
